uploader.json

```diff
@@ -2087,51 +2087,6 @@
             ],
             "symbolic_name": "nfs_exports_d"
         },
-        {
-            "file": "/etc/nginx/()*.conf",
-            "pattern": [],
-            "symbolic_name": "nginx_conf"
-        },
-        {
-            "file": "/etc/nginx/conf.d/()*",
-            "pattern": [],
-            "symbolic_name": "nginx_conf"
-        },
-        {
-            "file": "/etc/nginx/default.d/()*",
-            "pattern": [],
-            "symbolic_name": "nginx_conf"
-        },
-        {
-            "file": "/etc/opt/rh/rh-nginx()*/nginx/()*.conf",
-            "pattern": [],
-            "symbolic_name": "nginx_conf"
-        },
-        {
-            "file": "/etc/opt/rh/rh-nginx()*/nginx/conf.d/()*",
```

Comment:
Same change as above, remove the second set of `()`.

Reply:
@bfahr, here we can't remove the second (), since "*" can't appear single. We can keep it, I test it, it just match empty '' as bellow
```
>>> re.match('()*', 'a.conf')
<_sre.SRE_Match object at 0x10baaf648>
>>> re.match('()*', 'a.conf').group(0)
''
```
since our code just check if it is matched by
```
 if re.match(match, file_path):
```
so we don't need to update it.